boards/phytec/phycore_rt1170/phycore_rt1170_mimxrt1176_cm4_defconfig

```diff
@@ -0,0 +1,14 @@
+#
+# Copyright 2024, PHYTEC America, LLC
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+CONFIG_CONSOLE=y
+CONFIG_UART_CONSOLE=y
+CONFIG_SERIAL=y
+CONFIG_GPIO=y
+CONFIG_ARM_MPU=y
+CONFIG_HW_STACK_PROTECTION=y
+CONFIG_PINCTRL=y
```

Comment:
I think this CONFIG is not required too. I saw in another PR it's not allowed here.